ksqldb-rest-app/src/test/java/io/confluent/ksql/api/TlsTest.java

```diff
@@ -35,22 +35,20 @@
 import java.util.concurrent.TimeUnit;
 import org.apache.kafka.common.config.SslConfigs;
 import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-public class TlsTest extends ApiTest {
+public class TlsTest extends BaseApiTest {
```

Comment:
Hey @big-andy-coates , can we undo this change? It's intentional that TlsTest extends ApiTest so we have test coverage for issuing requests to a TLS-enabled server.

Reply:
Done in https://github.com/confluentinc/ksql/pull/6144